### DISCUSSION TITLE
Enable local device discovery after clikcing no on the prompt

### DISCUSSION THREAD

**Discussion by Hr46ph** [Eloston/ungoogled-chromium#3078] [Q&A]
I clicked no on the prompt when asked because my brain was turned off. Now I cannot access local websites anymore.

I searched everywhere, checked the chromium flags, looked through settings. Every website and chatgpt suggestion where the setting should be, I cannot find it.

Where is this hidden? 

Thanks.

**Comment by networkException** [comment]
I don't know about this prompt, what platform are you on?

**Comment by Hr46ph** [comment]
macOS. Could it be an OS prompt? I hadn’t thought about that one 🤔
